hack/release.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/sh
+#
+# Cut a libpod release.  Usage:
+#
+#   $ hack/release.sh
+
+DATE=$(date +%Y-%m-%d)
+MONTH=$(echo "${DATE}" | sed 's/[0-9]*-0\?\([0-9]*\)-[0-9]*/\1/')
+NEXT_MONTH=$(date --date 'next friday' +%m | sed 's/0\?\([0-9]*\)/\1/')
+LAST_TAG=$(git describe --tags --abbrev=0)
+LAST_MONTH=$(echo "${LAST_TAG}" | sed -n 's/v0\.\([0-9]*\)\.[0-9]*/\1/p')
+if test -z "${LAST_MONTH}"
+then
+	echo "unable to parse month from ${LAST_TAG}" >&1
+	exit 1
+fi
+LAST_COUNT=$(echo "${LAST_TAG}" | sed -n 's/v0\.[0-9]*\.\([0-9]*\)/\1/p')
+if test "${MONTH}" = "${LAST_MONTH}"
+then
+	COUNT=$(($LAST_COUNT + 1))
+else
+	COUNT=1
+fi
+if test "${MONTH}" = "${NEXT_MONTH}"
+then
+	NEXT_COUNT=$(($COUNT + 1))
+else
+	NEXT_COUNT=1
+fi
+VERSION="0.${MONTH}.${COUNT}"
+NEXT_VERSION="0.${NEXT_MONTH}.${NEXT_COUNT}"
+echo "cutting ${VERSION} (next release will be ${NEXT_VERSION})"
+
+write_go_version()
+{
+	LOCAL_VERSION="$1"
```

Comment:
If you are going to create a variable why not declare it local as well?

Reply:
> If you are going to create a variable why not declare it local as well?

Because `local` is a Bash-ism (or at least it's not [in POSIX][1]).  I'm also fine inlining these little helper functions if folks want, I just made them functions to keep the regexps away from everything else ;).

[1]: http://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html